src/views/components/package-sidemenu.html

```diff
@@ -6,18 +8,7 @@
 			<div class="sidemenu-title">
 				<span>Statistics</span>
 
-				<div class="btn-group">
-					<button type="button" class="btn dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-						<span>{{statsPeriod}}</span>
-						<i class="fa fa-angle-down" aria-hidden="true"></i>
-					</button>
-
-					<ul class="dropdown-menu">
-						{{#each statsPeriods}}
-							<li><a on-click="@this.set('statsPeriod', @key)">{{@key}}</a></li>
-						{{/each}}
-					</ul>
-				</div>
+				<c-periods-dropdown selectedValue="{{periodData.periodValue}}" selectHandler="{{@this.handlePeriodSelecting()}}"></c-periods-dropdown>
```

Comment:
Why do we need the select handler in addition to the selectedValue?

Reply:
Handler we need to get back value to the parent, selectedValue is also used as a default value for the dropdown. Somewhere we have quarter as a default, somewhere month.